types/sortable-options.d.ts

```diff
@@ -0,0 +1,204 @@
+	/**
+	 * ignore the HTML5 DnD behaviour and force the fallback to kick in
+	 */
+	forceFallback?: boolean;
+	/**
+	 * Class name for the drop placeholder
+	 */
+	ghostClass?: string;
+	/**
+	 * To drag elements from one list into another, both lists must have the same group value.
+	 * You can also define whether lists can give away, give and keep a copy (clone), and receive elements.
+	 */
+	group?: string | SortableGroupOptions;
+	/**
+	 * Drag handle selector within list items
+	 */
+	handle?: string;
+	ignore?: string;
+	/**
+	 * Threshold of the inverted swap zone (will be set to swapThreshold value by default)
+	 */
+	invertedSwapThreshold?: number,
+	/**
+	 * Will always use inverted swap zone if set to true
+	 */
+	invertSwap?: boolean,
+	/**
+	 * Call `event.preventDefault()` when triggered `filter`
+	 */
+	preventOnFilter?: boolean;
+	/**
+	 * Remove the clone element when it is not showing, rather than just hiding it
+	 */
+	removeCloneOnHide?: boolean;
+	scroll?: boolean;
+	/**
+	 * if you have custom scrollbar scrollFn may be used for autoscrolling
+	 */
+	scrollFn?: ((this: Sortable, offsetX: number, offsetY: number, event: MouseEvent) => void);
+	/**
+	 * px, how near the mouse must be to an edge to start scrolling.
+	 */
+	scrollSensitivity?: number;
+	/**
+	 * px
+	 */
+	scrollSpeed?: number;
+	/**
+	 * sorting inside list
+	 */
+	sort?: boolean;
+	store?: {
+		get: (sortable: Sortable) => string[];
+		set: (sortable: Sortable) => void;
+	};
+	/*
+	 * Threshold of the swap zone
+	 */
+	swapThreshold?: number,
+	/*
+	 * how many pixels the point should move before cancelling a delayed drag event
+	 */
+	touchStartThreshold?: number;
+	setData?: (dataTransfer: DataTransfer, draggedElement: HTMLElement) => void;
+	/**
+	 * Element dragging started
+	 */
+	onStart?: (event: SortableEvent) => void;
+	/**
+	 * Element dragging ended
+	 */
+	onEnd?: (event: SortableEvent) => void;
+	/**
+	 * Element is dropped into the list from another list
+	 */
+	onAdd?: (event: SortableEvent) => void;
+	/**
+	 * Created a clone of an element
+	 */
+	onClone?: (event: SortableEvent) => void;
+	/**
+	 * Dragging element changes position
+	 */
+	onChange?: (event: SortableEvent) => void;
+	/**
+	 * Element is chosen
+	 */
+	onChoose?: (event: SortableEvent) => void;
+	/**
+	 * Element is unchosen
+	 */
+	onUnchoose?: (event: SortableEvent) => void;
+	/**
+	 * Changed sorting within list
+	 */
+	onUpdate?: (event: SortableEvent) => void;
+	/**
+	 * Called by any change to the list (add / update / remove)
+	 */
+	onSort?: (event: SortableEvent) => void;
+	/**
+	 * Element is removed from the list into another list
+	 */
+	onRemove?: (event: SortableEvent) => void;
+	/**
+	 * Attempt to drag a filtered element
+	 */
+	onFilter?: (event: SortableEvent) => void;
+	/**
+	 * Event when you move an item in the list or between lists
+	 */
+	onMove?: (event: SortableMoveEvent) => boolean;
```

Comment:
Incorrect result and function parameters.

Should be this:
```ts
onMove?: (evt: MoveEvent, originalEvent: Event) => boolean | -1 | 1
```

Accoridng to the docs:
```js
// Event when you move an item in the list or between lists
	onMove: function (/**Event*/evt, /**Event*/originalEvent) {
		// Example: https://jsbin.com/nawahef/edit?js,output
		evt.dragged; // dragged HTMLElement
		evt.draggedRect; // DOMRect {left, top, right, bottom}
		evt.related; // HTMLElement on which have guided
		evt.relatedRect; // DOMRect
		evt.willInsertAfter; // Boolean that is true if Sortable will insert drag element after target by default
		originalEvent.clientY; // mouse position
		// return false; — for cancel
		// return -1; — insert before target
		// return 1; — insert after target
	},
```